tests/virt_control_plane_test.go

```diff
@@ -230,6 +230,49 @@ var _ = Describe("[Serial][ref_id:2717]KubeVirt control plane resilience", func(
 
 		})
 
+		When("[test_id:2807]Control plane pods temporarily lose connection to Kubernetes API", func() {
```

Comment:
We usually use `Describe`, `Context`, `It`. Let's keep it that way for consistency if there is no benefit.
We need to add [Serial] because this affects kubevirt health and it would cause other tests to fail.
I think Ginkgo shuffles test cases so these `It` would not run one after another. We need to use `BeforeEach` and `AfterEach` to setup and remove blackhole

Reply:
Just want to clarify here, you are saying the convention is to only have a single "When" in each "Context" - you want me to break this out into its own context?  Or that there should be no "When"? Trying to match convention within the existing test file.

I think I'll merge the recovery and just have a single 'It' case, if that is ok, to highlight that we want to have an explicit test of the recovery and it's not just a cleanup thing.

Reply:
Nevermind, seems we have used `When` in past, I apologize. 

>I think I'll merge the recovery and just have a single 'It' case, if that is ok, to highlight that we want to have an explicit test of the recovery and it's not just a cleanup thing.
Sounds good.